ui/pages/settings/networks-tab/networks-tab.js

```diff
@@ -92,7 +93,12 @@ const NetworksTab = ({ addNewNetwork }) => {
       {isFullScreen ? (
         <NetworksFormSubheader addNewNetwork={addNewNetwork} />
       ) : null}
-      <div className="networks-tab__content">
+      <div
+        className={classnames('networks-tab__content', {
+          'networks-tab__content--with-networks-list-popup-footer':
+            !isFullScreen && !shouldRenderNetworkForm,
+        })}
+      >
```

Comment:
Adds a conditional class that adds styles necessary to keep the content visible when a fixed footer is added and isolated the stacking context concerns to this div so the fixed footer will never overlap with any elements outside of `__content`

<img width="200" alt="Screen Shot 2022-06-15 at 8 40 58 AM" src="https://user-images.githubusercontent.com/8112138/173871206-2541a4a4-777b-4eb5-870d-2a13ba7e1c34.png">
<img width="200" alt="Screen Shot 2022-06-15 at 8 41 58 AM" src="https://user-images.githubusercontent.com/8112138/173871211-4c6fce84-477b-4f39-9eee-a38449af8003.png">